config/maintainers.json

```
@@ -113,7 +113,7 @@
     },
     {
       "github_username": "yyyc514",
```

```suggestion
      "github_username": "joshgoebel",
```

@joshgoebel as I understand you've renamed your account.

---

Correct. Thanks.